integration_tests/src/main/python/csv_test.py

```diff
@@ -618,15 +617,19 @@ def do_read(spark):
             cpu_fallback_class_name = cpu_scan_class,
             conf = conf)
     else:
-        assert_cpu_and_gpu_are_equal_collect_with_capture(
-            lambda spark: do_read(spark),
-            exist_classes = 'Gpu' + cpu_scan_class,
-            non_exist_classes = cpu_scan_class,
-            conf = conf)
+        if is_not_utc():  # non UTC is not support for csv, skip capture check
```

Comment:
Can we please file a follow on issue to comeback an look at this and the JSON case too?

Reply:
Yes, filed https://github.com/NVIDIA/spark-rapids/issues/9912 and https://github.com/NVIDIA/spark-rapids/issues/9913 to address this and updated code comments as well.